### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by blagoev**
Some updates to the readme instructions, etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -85,98 +89,87 @@ The Realm Flutter package name is `realm`
...
-        var objects = realm.query<Item>(r'make == $0', [mymake]);
-        ```
+        int price = 42;
+    }
+    ```
+
+* Generate RealmObject class `Item` from data model class `_Item`.
+
+    ```
+    dart run realm generate
```
Comment (nielsenko) [inline README.md:135]: Should we mention the `--watch` flag here?

File: README.md [hunk 2]
```diff
@@ -202,35 +195,16 @@ The Realm Dart package is `realm_dart`
...
     ``` 
 
-* Enable generation of Realm schema objects.
-
-    * Add `build_runner` package to `dev_dependencies`.
-
-        ```
-        dart pub add build_runner --dev
-        ```
-    * Enable realm_generator by adding a `build.yaml` file to the application.
```
Comment (nielsenko) [inline README.md:212]: Should we mention that the processed files can be configured with build.yaml?

**Review by nielsenko — APPROVED** [review]

File: README.md (hunk 1 above)
Comment (blagoev) [reply README.md:135]: This should probably be in the docs. this is only a getting started guide.

File: README.md (hunk 2 above)
Comment (blagoev) [reply README.md:212]: I think we could do that in the docs.

**Review by nielsenko — APPROVED** [review]

File: README.md [hunk 3]
```diff
@@ -85,106 +89,95 @@ The Realm Flutter package name is `realm`
...
 
-        // Get all objects
-        var items = realm.all<Item>();
-        
-        // Get object by index
-        var item = items[5];
-        
-        // Get object by primary key
-        var itemByKey = realm.find<Item>(0);
+    It should start with an underscore `_Item` and be annotated with `@RealmModel()`
```
Comment (desistefanova) [inline README.md:118]: or `$Item`

Comment (blagoev) [reply README.md:118]: This should be better documented in the docs.

File: README.md [hunk 4]
```diff
@@ -85,106 +89,95 @@ The Realm Flutter package name is `realm`
...
+    ```
+    A new file `catalog.g.dart` will be created next to the `catalog.dart`.
+    
+    _*This file should be committed to source control_
+
+* Use the RealmObject class `Item` with Realm.
+
+    ```dart
+    // Create a Configuration object
+    var config = Configuration(Item.schema);
```
Comment (desistefanova) [inline README.md:145]:
```suggestion
    var config = Configuration([Item.schema]);
```

Comment (blagoev) [reply README.md:145]: good catch. will fix

File: README.md [hunk 5]
```diff
@@ -85,106 +89,95 @@ The Realm Flutter package name is `realm`
...
+    // Get object by index
+    var item = items[5];
+    
+    // Get object by primary key
+    var itemByKey = realm.find<Item>(0);
+    
+    // Filter and sort object
+    var objects = realm.query<Item>("name == 'Special Item'");
+    var make = 'Tesla';
+    var objects = realm.query<Item>(r'make == $0', [make]);
```
Comment (desistefanova) [inline README.md:171]: Item doesn't have make property.

**Review by desistefanova — APPROVED** [review]

**Pull request merged**